modules/analysis/src/endpoints/test.rs

```diff
@@ -679,3 +679,59 @@ async fn spdx_package_of(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn spdx_ancestor_of(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let app = caller(ctx).await?;
+    ctx.ingest_documents(["spdx/1178.json"]).await?;
+
+    // This smells a little funny... are they backward?
+    let parent = "pkg:rpm/redhat/B@0.0.0";
+    let child = "pkg:generic/upstream-component@0.0.0?arch=src";
```

Comment:
@JimFuller-RedHat If it makes more sense, it's easy enough to switch these... [and these](https://github.com/trustification/trustify/blob/efc14daa5e187b50a6c7877eb4c85b52b6b61b39/modules/ingestor/src/graph/sbom/spdx.rs#L253)

Reply:
no these are correct (from a relationship point of view) ... I am trying to tease out this discussion elsewhere.